build.sbt

```diff
@@ -1325,7 +1286,7 @@ val createTargetClassesDir = taskKey[Unit]("create target classes dir")
 
 // Don't use these groups for any other projects
 lazy val sparkGroup = project
-  .aggregate(spark, sparkV1, sparkV1Filtered, sparkV2, contribs, sparkUnityCatalog, storage, storageS3DynamoDB, hudi, sharing)
+  .aggregate(spark, sparkV1, sparkV1Filtered, sparkV2, contribs, sparkUnityCatalog, storage, storageS3DynamoDB, hudi, sharing, connectCommon, connectClient, connectServer)
```

Comment:
Aside: is this `sparkGroup` helpful? What do _you_ think it means/represents/is used for?

e.g. should storage be in it?!

Reply:
Next question: who/what code uses `sparkGroup` // what's the benefit of adding more projects to it?